opcodes.c

```diff
@@ -174,8 +175,8 @@ static int inc_mem(struct em8051 *aCPU)
     if (address > 0x7f)
     {
         aCPU->mSFR[address - 0x80]++;
-        if (aCPU->sfrwrite)
-            aCPU->sfrwrite(aCPU, address);
+        if (aCPU->sfrwrite[address - 0x80])
+            aCPU->sfrwrite[address - 0x80](aCPU, address);
```

Comment:
still using the address - 0x80 in many places..?

Reply:
Oh, you want to have a temp variable everywhere ?

Reply:
In the end it doesn't really matter, I just figured it would be cleaner to do the subtract once and then use that everywhere. I know any optimizing compiler should(tm) figure that out and optimize it away, so the resulting code doesn't really change..

So anyway, you pick, either use the temp variable everywhere or nowhere. It's nice to see someone care about this code.

Reply:
> It's nice to see someone care about this code.

Well, **I really like it**, as it is one of the only ones that is curses based. 
It also has a nice divide between the "CPU simulator" and the surroundings. 

I'm actually growing to like 8051 more than AVR for learning purposes.
As it feels like a very good first stepping stone in the MCU world.

My learning curve is something like "Analog -> RTL Digital Logic -> IC Digital Logic -> 8051 -> AVR -> STM32".

So, thanks for it!

Reply:
> So anyway, you pick, either use the temp variable everywhere or nowhere. 

Then, it's going to be nowhere, as in the "don't fix something if not _obviously_ broken" 😉

Reply:
28d4ef90fb73e84e2e28679fda466612ac8884c1 it is then.

I opted for a `revert` instead of a `rebase` (to remove the commit) as IMHO it is also important to document why we do NOT do stuff (hence the lengthy commit message)